spec/factories/monthly_statistics_reports.rb

```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
+  factory :monthly_statistics_report, class: '::Publications::MonthlyStatistics::MonthlyStatisticsReport' do
+    month { '2023-12' }
+    generation_date { Time.zone.local(2023, 12, 18) }
+    publication_date { Time.zone.local(2023, 12, 25) }
+
+    trait :v1 do
+      statistics { V1MonthlyStatisticsStubbedReport.new.to_h }
+    end
+
+    trait :v2 do
```

Comment:
I create v1 and v2 to follow the convention in the code. If we plan to rename to v2024 and v2023 I will suggest to do in a separate PR.